src/common/vitals/vital-metric.js

```diff
@@ -0,0 +1,51 @@
+export class VitalMetric {
+  #values = []
+  #roundingMethod = Math.floor
+  #subscribers = new Set()
+
+  constructor (name, roundingMethod) {
+    this.name = name
+    this.attrs = {}
```

Comment:
Make these fields private. They should not be allowed to be changed outside this class. (I am assuming here)